crates/open_ai/src/open_ai.rs

```diff
@@ -473,7 +473,7 @@ pub async fn stream_completion(
             .filter_map(|line| async move {
                 match line {
                     Ok(line) => {
-                        let line = line.strip_prefix("data: ")?;
+                        let line = line.trim().strip_prefix("data:")?;
```

Comment:
Did you mean the trim to come after the strip? Otherwise we wouldn't strip the extra space. Also, I feel like we would only want to trim that one space so as not to cause other issues.

What provider was this? I am hesitant to merge this as it seems to be strange behavior from a provider